pkg/coordinator/deposits.go

```diff
@@ -179,7 +179,7 @@ func getDeposits(
 
 func printTable(deposits []depositEntry) error {
 	w := tabwriter.NewWriter(os.Stdout, 2, 4, 1, ' ', tabwriter.AlignRight)
-	fmt.Fprintf(w, "index\twallet\tvalue (BTC)\tdeposit key\tfunding transaction\tconfirmations\tswept\t\n")
+	fmt.Fprintf(w, "index\twallet\tvalue (BTC)\tdeposit key\trevealed deposit data\tconfirmations\tswept\t\n")
```

Comment:
Can we explicitly separate the "funding transaction" and "reveal block"? A value like `txHash:outputIndex:revealBlock` without additional explanation says nothing about what specific fields are.

Reply:
This is a tradeoff to simplify CLI usage as much as possible. The current approach requires a user to just copy-paste one property for each deposit from the table to the propose sweep CLI command without any modifications.

Reply:
Sure! Just flagging that this may not be obvious to someone not involved in the development of this CLI. We definitely need to establish the final audience so we know what tradeoffs are acceptable. cc @pdyraga

Reply:
I think that the current approach is fine. The target audience is devs in the early days until this is fully automated. And once that happens, that syntax is still acceptable given only devs will use the CLI in exceptional cases.